### ISSUE TITLE
Is this button background by design?

### ISSUE THREAD

**Issue by lolimay** [RocketChat/docs#1190]
![image](https://user-images.githubusercontent.com/32427260/55274810-8a3b3500-5317-11e9-9f3f-775ef8d11187.png)

I've noticed that there is a `#030c1a` background behind the <kbd>View Migration Docs
</kbd> button in [https://rocket.chat/atlassian](https://rocket.chat/atlassian).

Is this by design or a mistake?

**Comment by Rodriq** [comment]
@lolimay It's actually a design that went wrong.
  Inspect the element and if you can send a patch it'll be good.
Check it out lets see.

**Comment by lolimay** [comment]
> @lolimay It's actually a design that went wrong.
> Inspect the element and if you can send a patch it'll be good.
> Check it out lets see.

Yeah, but it can be on purpose possibly. I will try to make a patch for it and pull a request. Thanks anyway!
